### PR TITLE
fix: deletes all sessions for user on logout

### PR DESCRIPTION
**Pull request by chriswk**
To ensure we clean up sessions on logout this PR does two things:

1. When users changes password, it deletes all sessions for user
2. When logging out, it deletes all sessions for user, not just the current one in the request.

Will add a test before moving from draft

### CODE REVIEW

**Comment by chriswk** [comment]
tests added both for invalidating sessions when changing password and for logging out

File: src/lib/routes/logout.test.ts [hunk 1]
```diff
@@ -162,10 +228,61 @@ test('should redirect to alternative logoutUrl', async () => {
...
+            },
+        },
+        expired: addDays(new Date(), 2),
+    });
+    let activeSessionsBeforeLogout = await sessionStore.getSessionsForUser(1);
+    expect(activeSessionsBeforeLogout).toHaveLength(2);
+    app.use('/logout', new LogoutController(config, { sessionService }).router);
+    await supertest(app).get('/logout').expect(302);
+    let activeSessions = await sessionStore.getSessionsForUser(1);
+    expect(activeSessions).toHaveLength(0);
```
Comment (gardleopard) [inline src/lib/routes/logout.test.ts:287]: nice

File: src/test/e2e/api/admin/user-admin.e2e.test.ts [hunk 2]
```diff
@@ -231,11 +234,12 @@ test('validator should accept strong password', async () => {
...
 test('should change password', async () => {
     const user = await userStore.insert({ email: 'some@mail.com' });
-
-    return app.request
+    const spy = jest.spyOn(sessionStore, 'deleteSessionsForUser');
+    await app.request
         .post(`/api/admin/user-admin/${user.id}/change-password`)
         .send({ password: 'simple123-_ASsad' })
         .expect(200);
+    expect(spy).toHaveBeenCalled();
```
Comment (gardleopard) [inline src/test/e2e/api/admin/user-admin.e2e.test.ts:242]: nice:D

**Review by gardleopard — APPROVED** [review]
LGTM

**Pull request merged**